added Java resource

lovevictoria2020 opened the pull request:
Added a Java resource with tutorials, books, youtube channels and IDEs!

lovevictoria2020 committed:
added Java resource

lovevictoria2020 committed:
table of contents

lovevictoria2020 committed:
table of contents

Mayank57 commented:
everything seems fine except for the 2nd youtube link which should take me to the crash course but the link is for linear algebra course.

aditya-shirwatkar commented on programming-languages/README.md:
```diff
@@ -56,3 +80,33 @@ It is highly recommended to take the time and get familiar with the standard lib
+
+* [TheNewBoston Youtube Channel](https://www.youtube.com/channel/UCJbPGzawDH1njbqV-D5HqKw): This youtube channel is jam packed with tons of resources on learning Java. Are you a beginner? Start with the "Java (Beginner) Tutorial" playlist. When you feel you are ready, move onto to the "Java (Intermediate) Tutorial" playlist to learn some more advanced topics!
+
+* [Derek Banas]()https://www.youtube.com/channel/UCwRXb5dUK4cvsHbx-rGzSgw): Channel with crash course videos teaching you the basic of every programming language!
```
Remove the redundant ")" from "()https://.....". It is missing up the rich diff

lovevictoria2020 replied:
Check out my latest commit for the changes! Thanks so much!

aditya-shirwatkar commented on programming-languages/README.md:
```diff
@@ -56,3 +80,33 @@ It is highly recommended to take the time and get familiar with the standard lib
+### IDEs
+
+* [Eclipse](https://www.eclipse.org/downloads/): One of the most well known IDEs for Java. 
+* [JetBrains IntelliJ IDEA](https://www.jetbrains.com/idea/): A powerful IDE with an extensive library of features. You can get it for free for non-commercial projects as a student!
```
Replace "basic" with "basics"

lovevictoria2020 replied:
Fixed it! Check out my latest commit! Thanks so much for bringing that up!

lovevictoria2020 committed:
Made edits 

Hi! 

I made all the edits mentioned! Let me know what you think!

lovevictoria2020 commented:
> everything seems fine except for the 2nd youtube link which should take me to the crash course but the link is for linear algebra course.

Fixed it! It seems that he also had done a linear algebra course as well. Changed it so the link takes you the specific Java crash course! You can check my latest commit to see the edit!

take2rohit commented:
@lovevictoria2020 Thanks for your contribution. Do star and watch our other repos as well. 👍🏻
Happy Hacking. 😸

rishesh007 reviewed (commented):
The tutorials of TheNewBoston are very old, also he hadn't updated his lectures. Else all is fine👍.
